README.md

```diff
@@ -257,3 +257,8 @@ module.exports = {
     };
   }
 };
+
+
+## Getting pages tree
+
+In order to build a sitemap page, you can use the method `self.getPagesTree` from this module. It returns the nested pages and pieces pages in the right order. For each page you can access the array `_children` recursively to render the pages links at the right level.
```

Comment:
Needs an introduction:

"This module's primary purpose is creating a sitemap for Google and other search engines, but it is also useful in creating a sitemap for end users."

Reply:
"This method has a large performance impact each time it is called on a site with a large page tree, or many pieces reachable via pieces-pages. Strongly consider caching the response for a period of time."

Reply:
Done